spec/mailers/previews/overdue_training_alert_mailer_preview.rb

```diff
@@ -20,7 +20,7 @@ def example_alert_details
 
   def example_alert
     OverdueTrainingAlert.new(user: example_profile.user,
-                             course: ClassroomProgramCourse.current.nonprivate.last,
+                             course: Course.first,
```

Comment:
I think `Course.current.nonprivate.last` or `Course.nonprivate.last` would be better than `Course.first`. The latter should be just as reliable in production as using `Course.first`, as there will definitely be a least one nonprivate course on either of the production dashboards, while the first course might be private or more likely a test course or otherwise atypical one.

Reply:
Thank you, I will do that.